lib/rspec/rails/view_rendering.rb

```diff
@@ -61,6 +61,21 @@ def self.nullify_template_rendering(templates)
           end
         end
 
+        # @private
+        class LogSubscriber < ::ActiveSupport::LogSubscriber
+          def current_example_group
+            RSpec.current_example.example_group
+          end
+
+          def render_template(event)
+            unless current_example_group.render_views?
```

Comment:
Our rubocop configuration would like you to change this to

`return if current_example.render_views?` and then flatten the conditional. Could you do that?